src/constants/constants.js

```diff
@@ -780,20 +780,8 @@ export const CONTRACT_EVENT_FETCH_INTERVALS = {
     DEVELOPMENT: 4 * 1000,
 };
 
-export const BLOCKCHAIN_ID_TO_NAME = {
```

Comment:
I think we are still using this for backwards compatibility old and new ual format